examples/mautic-example-nginx-ssl-managedDB/README.md

```diff
@@ -0,0 +1,22 @@
+This example shows how to configure nginx to act as a reverse proxy with ssl support for mautic container.
+
+#### What is handled in this docker-compose.yml:
+
+* Setup of mysql and mautic containers
+* Setup of mysql db on managed or remote server
+* Setup of a nginx container with custom vhost configuration (nginx.conf)and secure with letsencrypt certs
+
+#### How to use this example:
+
+1. run ```docker-compose up``` in this directory
+2. add these lines to your /etc/hosts file 
+```127.0.0.1       pilotdev9.eastbayinfo.org ```
```

Comment:
We use [example.com in documentation](https://docs.mautic.org/en/contributing#example-domain-references) when referring to URLs - will refactor according to how I understand the URLs should be but feel free to correct if I make an error!

```suggestion
`127.0.0.1       mautic.example.com`
```